mappings/net/minecraft/block/BlockRenderLayer.mapping

```diff
@@ -1,2 +1,4 @@
 CLASS net/minecraft/class_1921 net/minecraft/block/BlockRenderLayer
 	FIELD field_9175 CUTOUT_MIPPED Lnet/minecraft/class_1921;
+	METHOD method_22723 preRender ()V
+	METHOD method_22724 postRender ()V
```

Comment:
Should probably rename these before this is merged too. These basically enter and exit rendering for each render layer